purchase_requisition_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# © 2016 ForgeFlow S.L. (https://www.forgeflow.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd.
```

Comment:
```suggestion
# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
```
Can you change that in all headers please?

Reply:
@AaronHForgeFlow Fixed it all.